app/views/referee_interface/reference/confidentiality.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :title, title_with_error_prefix(t('page_titles.referee.confidentiality', full_name: @application.full_name), @confidentiality_form.errors.any?) %>
+
+<% content_for :before_content, govuk_back_link_to(@previous_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+      model: @confidentiality_form,
+      url: referee_interface_confidentiality_path(token: @token_param, from: params[:from]),
+      method: :patch,
+    ) do |f| %>
+
+      <%= f.govuk_error_summary %>
+
+      <%= f.govuk_radio_buttons_fieldset :confidential, legend: { size: 'l', tag: 'h1', text: t('referee.confidentiality.choice.label', full_name: @application.full_name) } do %>
+        <%= f.govuk_radio_button :confidential, false, label: { text: t('referee.confidentiality.choice.yes') }, link_errors: true %>
+        <%= f.govuk_radio_button :confidential, true, label: { text: t('referee.confidentiality.choice.no') } %>
+      <% end %>
+
+      <%= f.govuk_submit t('continue') %>
```

Comment:
All the other bits of the reference form are 'save and continue'. Is there a reason why this is just continue?